plugins/unbound/unbound_

```diff
@@ -0,0 +1,535 @@
+#!/bin/sh
+# -*- sh -*-
+
+: << =cut
+
+=head1 NAME
+
+unbound_ - Munin plugin to monitor the Unbound DNS resolver.
+
+=head1 APPLICABLE SYSTEMS
+
+System with unbound daemon.
+
+=head1 CONFIGURATION
+
+Environment variables for this script
+
+	unbound_conf	- where the unbound.conf file is located.
+	unbound_control	- where to find unbound-control executable.
+	spoof_warn	- what level to warn about spoofing
+	spoof_crit	- what level to crit about spoofing
+
+You can set them in your munin/plugin-conf.d/plugins.conf file
+with:
+
+  [unbound_*]
+  user root
+  env.unbound_conf /etc/unbound/unbound.conf
+  env.unbound_control /usr/sbin/unbound-control
+  env.spoof_warn 1000
+  env.spoof_crit 100000
+
+Use the .env settings to override the defaults.
+
+To install; enable statistics and unbound-control in unbound.conf
+	server:		extended-statistics: yes
+			statistics-cumulative: no
+			statistics-interval: 0
+	remote-control:	control-enable: yes
+Run the command unbound-control-setup to generate the key files.
+
+=head1 USAGE
+
+This plugin can create different graphs depending on what name you link
+it as (with ln -s) into the plugins directory. You can link it multiple
+times. If you are only a casual user, the _hits and _by_type are most
+interesting, possibly followed by _by_rcode.
+
+unbound_hits		- base volume, cache hits, unwanted traffic
+unbound_queue		- to monitor the internal requestlist
+unbound_memory		- memory usage
+unbound_by_type		- incoming queries by type
+unbound_by_class	- incoming queries by class
+unbound_by_opcode	- incoming queries by opcode
+unbound_by_rcode	- answers by rcode, validation status
+unbound_by_flags	- incoming queries by flags
+unbound_histogram	- histogram of query resolving times
+
+=head1 AUTHOR
+
+Copyright 2008 W.C.A. Wijngaards
+          2013 Kim B. Heino
+
+=head1 LICENSE
+
+BSD
```

Comment:
Maybe you could add a more specific [SPDX license key](https://spdx.org/licenses/)?